library/dash/src/main/java/com/google/android/exoplayer2/source/dash/manifest/DashManifestParser.java

```diff
@@ -901,6 +903,33 @@ protected static String parseString(XmlPullParser xpp, String name, String defau
     return value == null ? defaultValue : value;
   }
 
+  protected static int parseDolbyChannelConfiguration(XmlPullParser xpp, String name,
+      int defaultValue) {
+    String value = xpp.getAttributeValue(null, name).toUpperCase();
```

Comment:
This probably gives a lint warning, because toUpperCase behaves differently depending on the phone's locale. I'd suggest using `Util.toLowerInvariant` instead, which will convert to lower case explicitly using the US English locale to make sure it behaves the same regardless of the phone locale. Obviously you'd need to make the constants below lowercase too. Alternatively, you can use toUpperCase(Locale.US) here.